packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6361,3 +6861,214 @@ Property details
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- |
+| lifecycleCreate            | setting that determines whether the updates to a user in Okta will be update a user in the application   | [Lifecycle Create Setting Object](##lifecycle-create-setting-object)      | TRUE    | FALSE   | FALSE    |           |
+
+###### Update object
+
+The Create object is composed of multiple settings that determine whether the updates to a user in Okta will update a user in the application.
+
+```json
+{
+  "profile": {
+      "status": "DISABLED"
+  },
+  "lifecycleDeactivate": {
+      "status": "DISABLED"
+  },
+  "password": {
+      "status": "DISABLED",
+      "seed": "RANDOM",
+      "change": "KEEP_EXISTING"
+  }
+}
+```
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- |
+| lifecycleDeactivate           | setting that determines whether deprovisioning will occur when app is unassigned  | [Lifecycle Deactivate Setting Object](#lifecycle-deactivate-setting-object)   | TRUE    | FALSE   | FALSE    |
+| password           | setting that determines whether Okta creates and pushes a password in the application for each assigned user | [Password Setting Object](#password-setting-object)    | TRUE    | FALSE   | FALSE    |
+| profile           | setting that determines whether the updates to a user in Okta will be update a user in the application.     | [Profile Setting Object](#profile-setting-object)     | TRUE    | FALSE   | FALSE    |
+
+###### Lifecycle Create Setting object
+
+Creates or links a user in the application when assigning the app to a user in Okta.
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly | Default |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- |
+| status   | status of the setting     | `ENABLED`, `DISABLED` | FALSE    | FALSE   | FALSE    | `DISABLED` |
+
+###### Lifecycle Deactivate Setting object
+
+Deactivates a user's application account when it is unassigned in Okta or their Okta account is deactivated. Accounts can be reactivated if the app is reassigned to a user in Okta.
```

Comment:
Deactivates a user's application account when it is unassigned in Okta or **if** their Okta account is deactivated. Accounts can be reactivated if the app is reassigned to a user in Okta.